ops/packer/build_bfd-platinum.json

```diff
@@ -18,10 +18,18 @@
           "tag:Name": "bfd-mgmt-vpn-private"
         }
       },
-      "ami_name": "bfd-platinum-{{isotime \"20060102030405\"}}",
+      "ami_name": "bfd-amzn2-platinum-{{isotime \"20060102030405\"}}",
       "ssh_pty": true,
+      "launch_block_device_mappings": [
+        {
+          "device_name": "/dev/xvda",
+          "volume_type": "gp2",
+          "volume_size": 30,
+          "delete_on_termination": true
+        }
+      ],
```

Comment:
Ensure our platinum image has the same root volume size as RHEL 7 gold image did.